### ISSUE TITLE
Labelling: Add new labelling rule

### ISSUE THREAD

**Issue by bfrgottscma** [SiLeBAT/fcl-client#569]
As a user I would like to add new labelling rules.

When clicking the "+" button under the list of labelling rules, a dialogue opens (Mockup: as for already existing labelling rules but without any entries).

Mandatory:
- user enters name for rule (free text)
- user defines conditions (either manually or automatically via the "add selection" button). 

Conditions dialogue:
- Property values and operations should be the same as used for the highlighting function. 
- For the values, offer the values that are true for the respective property. 
- For the Row menu allow and/or/remove. When remove is selected, the rule is deleted, or if it is the last remaining row, delete the values for this row.

Apply --> condition is applied
Cancel --> dialogue closed
OK --> condition is applied, dialogue closed

When labelling dialogue is open, clicking on another condition is blocked --> you have to click Apply/Cancel/OK before.

**Think of Tests**
- [ ] Excel testsheet needed
- [ ] E2e test needed
- [ ] Integration test needed
- [ ] Unit test needed

**Additional context**
- [ ] Mockup needed

**General Guidelines**
- Describe only one atomic unit per ticket
- Stay in the ticket scope! Be patient and write a new ticket in case of further feature requests.
- Talk with product owner if something might change for her/him.
- Improve ticket during grooming / split ticket / talk
- Do not groom / estimate without clear concept
- Tickets should be in the QA lane 24hrs before the sprint ends
- Every morning have a look whether code needs to be reviewed and directly do the review OR give feedback when you intend to do so

**Comment by afalenski** [comment]
- When labelling dialogue is open, clicking on conditions is always possible, also adding or deleting several conditions. This is as it should be in my opinion. @bfrgottscma What do you mean with "When labelling dialogue is open, clicking on another condition is blocked"?
- When labelling dialogue is open, clicking on other labels in the list is not possible - only after clicking Cancel or OK.
- Buttons Apply, Cancel and OK word as expected.
- Label dialogue is organised as discussed.
- [ ] In Chrome I see a scrollbar next to "Use conditions".
![image.png](https://images.zenhubusercontent.com/5a854eea4b5806bc2bc70715/bed1b277-e062-45ca-8632-a8ca92db7742) 
In Firefox it looks like this, which is good:
![image.png](https://images.zenhubusercontent.com/5a854eea4b5806bc2bc70715/180e5c98-fa14-4f0f-8e37-ea48e4b9e8bd)
**Please remove scrollbar.**
- The Add Selection button works fine.
- Adding Conditions, connecting with AND or OR, deleting condition and deleting content of the only condition row left work fine.
- Condition values from dataset are offered.
______
**Tested with**
OS: Windows 10

Datasets: 
- Example Dataset

Browsers:
- Google Chrome Version 92.0.4515.159 (Offizieller Build) (64-Bit)
- Firefox Quantum 68.9.0esr (64-Bit)

**Comment by ruegenbfr** [comment]
Ripple was removed for checkbox(es) in label edit dialogue and color/shape edit dialogue.

**Comment by bfrgottscma** [comment]
Everything as described. Done
